components/Styles/StyledStatsContainer.js

```diff
@@ -1,19 +1,12 @@
 import styled from "styled-components";
 
 const StyledStatsContainer = styled.ul`
-  display: flex;
-  justify-content: center;
-  flex-direction: column;
-  flex-wrap: wrap;
-  border: black solid 2px;
-  border-radius: 5px;
-  height: 37vh;
-  margin: 1rem;
+display: grid;
```

Comment:
One tipp regarding grid - we've had some issues with the position of the grid on the page and found out, that per default there is a ```padding-inline-start: 40px```defined. Maybe you set this to 0px as well, to avoid this placing issue: 

![Bildschirmfoto 2024-05-30 um 10 59 08](https://github.com/Braun-Johannes/Capstone-Project-BattlePets/assets/164749128/304c7beb-4d6e-4dfd-a481-533372c7f2d0)

Reply:
Thanks for the example. We will look for this as well while we clean up the css parts of our code.